quipucords/quipucords/featureflag.py

```diff
@@ -0,0 +1,49 @@
+"""Filters feature flags from systen env variables."""
+
+import logging
+import os
+
+DEFAULT_FEATURE_FLAGS_VALUES = {
+    'OVERALL_STATUS': False}  # default values for feature flags that
+# are already implemented
+
+VALID_VALUES_FOR_ENV_VARIABLES = ['0', '1']
+
+# Get an instance of a logger
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def get_feature_flags_from_env():
+    """Filter feature flags from environment variables."""
+    global DEFAULT_FEATURE_FLAGS_VALUES
+    global VALID_VALUES_FOR_ENV_VARIABLES
+    for key, value in os.environ.items():
+        if key.upper().startswith('QPC_FEATURE_'):
+            feature = key.upper().replace('QPC_FEATURE_', '')
+            try:
+                if value in VALID_VALUES_FOR_ENV_VARIABLES:
+                    DEFAULT_FEATURE_FLAGS_VALUES[feature] = bool(int(value))
+            except ValueError:
+                logger.info(' %s can`t be converted to int, verify your '
+                            'environment variables.', value)
+                continue
+    return DEFAULT_FEATURE_FLAGS_VALUES
+
+
+class FeatureFlag:
+    """Handle environment variables and its status."""
+
+    def __init__(self, initial_data):
```

Comment:
This is optional: I'd change how `__init__` handles initial data. Instead of expecting `initial_data` it could either

1. replace it with "parser function" that returns the dict with (pointing as default to `get_feature_flags_from_env`)
2. refactor `get_feature_flags_from_env` as a `classmethod`* that would be called inside `__init__` 

 On either case `initial_data` would be set inside `__init__` calling either the parser function or the proper method.